tests/ClientTest.php

```diff
@@ -466,4 +520,17 @@ protected function getResponseMock($body, $status, $defaultHeaders = true)
 
         return $responseMock;
     }
-}
\ No newline at end of file
+
+    /**
+     * @param array|null $methods
+     * @return \HTTP_Request2|MockObject
+     * @throws \ReflectionException
+     */
+    protected function getRequestMock($methods = null)
+    {
+        return $this->getMockBuilder('\HTTP_Request2')
```

Comment:
Instead of using this string:
* Import the class.
* Use `HTTP_Request2::class` instead.